test/cucumber/features/cluster-mode/kafka-packages.cluster.feature

```diff
@@ -18,7 +18,6 @@ Feature: Kafka and its dependent packages?
         When I launch the platform with params
         Then The service "grafana" should be started with 1 replica
         And The service "prometheus" should be started with 1 replica
-        And The service "prometheus-kafka-adapter" should be started with 1 replica
         And The service "prometheus_backup" should be started with 1 replica
```

Comment:
I think you forgot to remove the monitoring test from the cluster as well that's why the tests are failing

Reply:
Ah I see, thanks.